core/cmd/testdata/vrf_password.txt

```diff
@@ -0,0 +1 @@
+testingpassword
```

Comment:
This reminds me that VRF is the one key type that hasn't been harmonized with the rest (noticed that the password doesn't fulfill our new strength requirements)

Reply:
We kind of need to rethink all of our keys honestly since decrypting is so damn expensive.

Ideally we'd have one encrypted keystore which holds everything.

Reply:
Alternatively we can reduce scrypt difficulty which I know @kaleofduty will love :trollface:

Reply:
> Ideally we'd have one encrypted keystore which holds everything.

Agreed, it would be nice if a single keystore could hold multiple different key types.


> Alternatively we can reduce scrypt difficulty which I know @kaleofduty will love 

How about we replace scrypt with md5? I remember that being recommended by a PHP tutorial from 300 B.C.  :trollface: